apps/console/src/features/organizations/components/organization-attributes.tsx

```diff
@@ -0,0 +1,184 @@
+
+    /**
+     * Is Read Only access
+     */
+    isReadOnly: boolean;
+
+    /**
+     * On Attribute Update callback
+     */
+    onAttributeUpdate: (orgId: string) => void
+}
+
+export const OrganizationAttributes: FunctionComponent<OrganizationAttributesPropsInterface> = (
+    props: OrganizationAttributesPropsInterface
+): ReactElement => {
+
+    const {
+        organization,
+        isReadOnly,
+        onAttributeUpdate,
+        ["data-testid"]: testId
+    } = props;
+
+    const { t } = useTranslation();
+    const dispatch = useDispatch();
+
+    const [ submit, setSubmit ] = useTrigger();
+    const [ isSubmitting, setIsSubmitting ] = useState(false);
+
+    const updateOrgAttributes = useCallback((data: KeyValue[]) => {
+        setIsSubmitting(true);
+        const attributes = organization.attributes;
+
+        const updatedAttributes: PatchData[] = attributes
+            .map((attribute) => data.find((updated) => updated.key === attribute.key))
+            .filter((updated) => updated !== undefined)
+            .map((updated) => ({
+                operation: "REPLACE",
+                path: `/attributes/${updated.key}`,
+                value: updated.value
+            }));
+
+        const addedAttributes: PatchData[] = data
+            .filter((updated) => attributes.findIndex((attribute) => attribute.key === updated.key) === -1)
+            .map((attribute) => ({
+                operation: "ADD",
+                path: `/attributes/${attribute.key}`,
+                value: attribute.value
+            }));
+
+        const removedAttributes: PatchData[] = attributes
+            .filter((attribute) => data.findIndex((updated) => updated?.key === attribute?.key) === -1)
+            .map((attribute) => ({
+                operation: "REMOVE",
+                path: `/attributes/${attribute?.key}`,
+                value: attribute.value ?? ""
+            }));
+
+        const patchData: PatchData[] = [
+            ...updatedAttributes,
+            ...addedAttributes,
+            ...removedAttributes
+        ];
+
+        patchOrganization(organization.id, patchData)
+            .then((_response) => {
+                dispatch(
+                    addAlert({
+                        description: t(
+                            "console:manage.features.organizations.notifications.updateOrganizationAttributes." +
+                            "success.description"
+                        ),
+                        level: AlertLevels.SUCCESS,
+                        message: t(
+                            "console:manage.features.organizations.notifications.updateOrganizationAttributes." +
+                            "success.message"
+                        )
+                    })
+                );
+
+                onAttributeUpdate(organization.id);
+            }).catch((error) => {
+                if (error.response && error.response.data && error.response.data.description) {
```

Comment:
Since we are returning `error.response.data` from the API method, we can straightaway use the error object here.